.github/workflows/maven.yml

```
@@ -2,23 +1,76 @@
+                  PGPASSWORD: ${{ env.SPRING_DATASOURCE_PASSWORD }}
 
-    steps:
-      - uses: actions/checkout@v4
-      - name: Set up JDK 21
-        uses: actions/setup-java@v3
-        with:
-          java-version: '21'
-          distribution: 'temurin'
-          cache: maven
+            - name: Set up JDK 21
+              uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)